README.md

```diff
@@ -49,16 +49,17 @@ This extension will help you work with dbt and BigQuery.
 
 ## How to use
 
-Extension works on macOS, now it supports the default locations for `profiles.yml` and `dbt_project.yml` files and the BigQuery destination with [OAuth via gcloud](https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile#oauth-via-gcloud), [Service Account File](https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile#service-account-file) and [Service Account JSON](https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile#service-account-json) authorization types.
+Extension works on MacOS, Linux and Windows (for Windows see the additional steps in [Windows support](docs/WindowsSupport.md) article). It supports the default locations for `profiles.yml` and `dbt_project.yml` files and the BigQuery destination.
```

Comment:
`BigQuery` is not a destination in terms of dbt. It's something like source or Profile. See their [docs](https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile).

Reply:
Good point, I will rename it to Profile.